### ISSUE TITLE
[Tech question] GPU usage in 60Hz vs 120Hz+VsyncSwapPeriod=2

### ISSUE THREAD

**Issue by vanfanel** [libretro/mupen64plus-libretro-nx#558]
Hello there,

My iGPU (Intel XE on an i5-1235U) doesn't provide VRR over HDMI (it's 2.0 only), so to prevent input lag caused by VSYNC, I run most LibRetro cores on a 120Hz mode with Vsync Swap Period set to 2.
That's supposed to tax the GPU the same as 60Hz with Vsync Swap Period set to 1, not more, but I am seeing that some games (Conquer, for example) are showing performance problems with 120Hz+Vsync Swap Period set to 2, while they are fullspeed at 60Hz with Vsync Swap Period set to 1.

This is something I am only seeing on this particular core, and only with the ParaLLEI graphics plugin.
Is it a known issue? Is this plugin somehow hammering the GPU at 120Hz even if Vsync Swap Period is set to 2?

Thanks!

**Comment by golivax** [comment]
Similar situation here. I can't get a fully stable 60fps on Goldeneye and Cruisn USA on an Rog Ally if I'm running  120Hz + Sync to Exact Content Framerate (Vsync off) with Mupen64plus Next and Parallel plugin. It seems to tax the GPU like crazy for whatever reason.

**Comment by vanfanel** [comment]
Yes, but it's obviously wrong: 120Hz + Vsync Swap Period = 2 should result in the same number of frames being rendered as 60Hz + Vsync Swap Period = 1.

**Comment by golivax** [comment]
@vanfanel have you discovered any workaround for this? In my case, only way to get perfect frame pacing is with Sync to Exact Content Framerate on, but then I have slowdowns in many games (Goldeneye, Cruisin USA). I tried RetroArch v1.20, but the problem is still there. Feels like it is either an issue with this core or ParallelRDP.

**Comment by vanfanel** [comment]
> [@vanfanel](https://github.com/vanfanel) have you discovered any workaround for this? In my case, only way to get perfect frame pacing is with Sync to Exact Content Framerate on, but then I have slowdowns in many games (Goldeneye, Cruisin USA). I tried RetroArch v1.20, but the problem is still there. Feels like it is either an issue with this core or ParallelRDP.

Hello @golivax 

No, sadly I couldn't find any workaround :(
